seshdash/views.py

```diff
@@ -796,62 +797,81 @@ def historical_data(request):
         context_dict['sort_dict'] = sort_data_dict
         return render(request, 'seshdash/data_analysis/historical-data.html', context_dict);
 
-#function for Graph Generations
+
+@require_GET
 @login_required
-def graphs(request):
+def graphs(request): 
+    """
+    Returns json, containing data that is used in data analysis graphs
+    """
+    results = []
 
-    # if ajax request
-    if request.method == 'POST':
+    # Getting values from Post request
+    time = request.GET.get('time', '') # This is the time range it has to be: 24h, 7d or 30d
+    choices = request.GET.getlist('choice[]')
+    active_id = request.GET.get('active_site_id', None)
+    start_time = request.GET.get('start_time', None)
+    end_time = request.GET.get('end_time', None)
```

Comment:
default for end time needs to be now. We shouldn't be able to request an endtime in the future as well